.github/workflows/build-deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - main
+
+env:
+ CI: false
```

Comment:
From what I understand you wanted to prevent treating a warning as an error in the build process. What do you think then about moving it to this particular job?

Reply:
TBH it is a leftover. IMO it can be removed.

Reply:
I removed it and build failed because of vulnerabilities I'm not able to resolved right now. I'll remove the flag to install and biuld step as you suggest.